rtbkit/core/router/router_runner.cc

```diff
@@ -151,6 +154,8 @@ init()
             << "slow-mode-money-limit= " << amountSlowModeMoneyLimit <<endl;
     }
 
+    Seconds augmentationWindow = std::chrono::milliseconds(5);
```

Comment:
this doesn't take into account the parameter passed in as an argument to the router_runner. This will always set to 5ms.